tests/data/ch08/main/count_syllables.txt

```diff
@@ -0,0 +1,15 @@
+Not found: yggdrasil
```

Comment:
I'd love to see a phoneme list for `yggdrasil`. In the meantime, I'd now be able to add it to `missing_words.json` as `"yggdrasil": 3`, which is handy.